entries/l/linktr.ee.json

```
@@ -1,10 +1,10 @@
 {
   "Linktree": {
     "domain": "linktr.ee",
-    "contact": {
-      "facebook": "Linktree",
-      "twitter": "Linktree_"
-    },
+    "tfa": [
+      "sms"
```

According to their docs, "voice call" is supported as well.
```suggestion
      "sms",
      "call"
```